po/en.po

```diff
@@ -7445,7 +7445,7 @@ msgstr ""
 msgid ""
 "You have performed %(x_nb1)s searches (%(x_nb2)s different questions) during "
 "the last 30 days or so."
-msgstr ""
+msgstr "You have performed %(x_nb1)s searches (%(x_nb2)s different questions)."
```

Comment:
This should be better fixed by amending the original English source code string, so that other I18N translations won't get out of sync.